.gitignore

```diff
@@ -2,9 +2,20 @@
 flare.tar.gz
 venv
 venv-tools
-flare/local
-flare/metadata/local.meta
+packages/flare/local
```

Comment:
packages? What's happening here?

Reply:
The Splunk NPX scaffolding uses lerna and their base folder for the packages is called packages